src/SFA.DAS.ContentApi.Database/Tables/Content.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[Content]
+(
+	[Id] INT NOT NULL PRIMARY KEY, 
+    [ContentTypeId] INT NOT NULL, 
+    [Content] TEXT NULL, 
+    [StartDate] DATETIME NOT NULL, 
```

Comment:
There's no requirement for these dates at the moment so we should not include them initially.

Reply:
Need Corey to confirm but I think they will be used